src/psyclone/tests/psyir/nodes/node_test.py

```diff
@@ -735,6 +735,13 @@ def test_find_or_create_symbol():
             "find_or_create_symbol method, is not an ancestor of this node "
             "'Reference[name:'alpha']'." in str(excinfo.value))
 
+    # find_or_create_symbol method with invalid visability
```

Comment:
s/visability/visibility/

Reply:
Thanks. I find that word hard to spell :-)